config/checkstyle_checks.xml

```diff
@@ -447,7 +447,7 @@
     <!-- Size Violations -->
     <module name="AnonInnerLength"/>
     <module name="ExecutableStatementCount">
-        <property name="max" value="30"/>
+        <property name="max" value="31"/>
```

Comment:
it is linked with [work-around](https://github.com/checkstyle/checkstyle/pull/4832/files#r129677226) caused by [pitest strange behavior](https://github.com/checkstyle/checkstyle/issues/4832)